app/views/storytime/dashboard/posts/_list.html.erb

```diff
@@ -3,7 +3,12 @@
     <%= link_to [:edit, :dashboard, post], id: "#{post.class.to_s.downcase.split("::").last}_#{post.id}", class: "list-group-item" do %>
       <div class="row">
         <div class="col-md-8">
-           <h4><%= post.title %></h4>
+          <h4><%= post.title %></h4>
+        </div>
+        <div class="col-md-2 col-md-offset-2">
+          <%= form_tag(url_for([:dashboard, post]), method: :delete) do |f| %>
```

Comment:
@dvanderbeek 
I tried without js, but as you can see, in  current markup we must use delete link in another link(editing post) and that's why i rejected from this idea.
Ok, if we must use %ul, this markup(whole view) can be similar to that you described, I will try to fix it.